test/module-enabled/variables.tf

```diff
@@ -0,0 +1,17 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# UNIT TEST MODULE
+# This module exists for the purpose of testing only and should not be
+# considered as an example.
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+variable "aws_region" {
+  description = "The AWS region in which all resources will be created."
+  type        = string
+  default     = "us-east-1"
```

Comment:
counterintuitive - see comment about the random region selection in the code.

Reply:
It's a test and not an example. If you'd like to change the way we are using regions in our tests, please provide a separate pr that we can discuss.

Reply:
as set this example should be discussed in the template directory.